tools/devops/device-tests/README.md

```diff
@@ -73,6 +73,68 @@ The are two main groups of variables used in the pipelines:
 - common (under templates/common-variables.yml): Contains a set of group variables that have been created by release engineering and that contain all the different secrets needed by the pipeline.
 - lab (under templates/[ddfun|cambridge]-variables: Contains the different configurations per lab. That includes the pools to use and the type of storage.
 
+There are vertain enviroment variables that are created by steps in the first job that are later shared with the second job. In order to do that you
+have to ensrue that you do the following:
+
+```yaml
+- bash: |
+    set -x
+    set -e
+    echo 'Use xamarin storage $USE_XAMARIN_STORAGE'
+
+    cd $WORKING_DIR/xamarin-macios
+    if [[ "$USE_XAMARIN_STORAGE" == "True" ]]; then
+      EC=0
+      ssh builder@xamarin-storage "mkdir -p /volume1/storage/$XAMARIN_STORAGE_PATH" || EC=$?
+      if [ $EC -eq 0 ]; then
+        echo '##vso[task.setvariable variable=XAMARIN_STORAGE_FAILED;isOutput=true]false'
+        echo '##vso[task.setvariable variable=XAMARIN_STORAGE_PATH;isOutput=true]$XAMARIN_STORAGE_PATH'
+        export TESTS_PERIODIC_COMMAND="--periodic-interval 10 --periodic-command rsync --periodic-command-arguments '-avz -e \"ssh\" $PWD/jenkins-results builder@xamarin-storage:/volume1/storage/$XAMARIN_STORAGE_PATH'"
+      else
+        echo '##vso[task.setvariable variable=XAMARIN_STORAGE_FAILED;isOutput=true]true'
+        echo '##vso[task.setvariable variable=XAMARIN_STORAGE_PATH;isOutput=true]""'
+      fi
+    fi
+
+    make -C builds download -j || true
+    make -C builds downloads -j || true
+    make -C builds .stamp-mono-ios-sdk-destdir -j || true
```

Comment:
This can be simplified now that this code is in our own repo (the `downloads` target doesn't exist anymore, but it was still in Azure Devops because the pipeline could be used to build old commits).

```suggestion
    make -C builds download .stamp-mono-ios-sdk-destdir -j
```